.github/workflows/deploy.yml

```diff
@@ -0,0 +1,52 @@
+name: Publish packages
+
+on:
+  push:
+    branches: [ master ]
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: ['3.6']
+    
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        # Include tags in the checkout
+        fetch-depth: 0
```

Comment:
Ha, that's awesome

Reply:
Glad we don't have any big files in our history!